packages/ui/components/ProteinForm/ProteinForm.tsx

```diff
@@ -1,86 +1,125 @@
+import styles from './proteinform.module.css';
+import { Form, Input, Button } from 'antd';
 
-function ProteinForm({
+const ProteinForm = ({
   setBasicInfo,
   setShowMutationForm,
   setShowProteinForm,
   setShowEligibleSection,
   showProteinForm,
-}: any) {
+  setCurrent,
+}: any) => {
+  const layout = {
+    labelCol: { span: 8 },
+    wrapperCol: { span: 16 },
+  };
+  const tailLayout = {
+    wrapperCol: { offset: 8, span: 16 },
+  };
+
+  const Demo = () => {
+    const onFinish = (values: any) => {
+      console.log('Success:', values);
+    };
+  };
+
+  const onFinishFailed = (errorInfo: any) => {
+    console.log('Failed:', errorInfo);
+  };
   return (
-    <div className="flex justify-between ">
-      <div className="w-3/5">
-        <h1 className="mt-6 text-2xl font-normal">
-          What is the size of the protein?
-        </h1>
-        <form className="w-full">
-          <div className="mt-4">
-            <label
-              htmlFor="less-amino"
-              className="ml-1 text-sm flex items-center"
-            >
-              <input type="radio" id="less-amino" className="mr-2" />
-              <Icon
-                name="chevron-left"
-                className="w-4 ml-1 text-black"
-              />
-              1100 amino acids
-            </label>
-            <br />
-          </div>
-          <div className="mt-4">
-            <label
-              htmlFor="more-amino"
-              className="ml-1 text-sm flex items-center"
-            >
-              <input type="radio" id="more-amino" className="mr-2" />
-              <Icon
-                name="chevron-right"
-                className="w-4 ml-1 text-black"
-              />{' '}
-              1100 amino acids
-            </label>
-            <br />
-          </div>
-          <div className="mt-4">
-            <label
-              htmlFor="4"
-              className="ml-1 text-sm flex items-center"
-            >
-              <input type="radio" className="mr-4" />
-              Dont Know
-            </label>
-          </div>
-          <div className="flex mt-4">
+    <div className={styles['protein-form']}>
+      <div style={{ width: '60%' }}>
+        <h1>What is the size of the protein?</h1>
+        <Form
+          name="basic"
+          initialValues={{ remember: true }}
+          onFinish={Demo}
+          onFinishFailed={onFinishFailed}
+        >
+          <Form.Item
+            htmlFor="less-amino"
+            style={{
+              fontSize: 'small',
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'center',
+            }}
+          >
+            <Input
+              type="radio"
+              id="less-amino"
+              style={{ marginRight: '0.5rem' }}
+            />
+            {/* <Icon
+              name="chevron-left"
+              className={styles['icon-left']}
+            /> */}
+            1100 amino acids
```

Comment:
The design says "< 1100 amino acids", but "Less than 1,100 amino acids" is more accessible to most people. Also, @sanathkr the next option in the design says "> 1100" but does one of these need to be "Less than or equal to" or "Greater than or equal to" ?

Reply:
Good point. Let's do 
* Less than or equal to 1,100 amino acids
* Greater than 1,100 amino acids